### PR TITLE
ENG-1070: Fix false positive

### PR DESCRIPTION
**Pull request by w-caffiero-entando**

### CODE REVIEW

**Review by ampie — APPROVED** [review]
/lgtm

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-plugin-jpseo/pull/12#pullrequestreview-489748589" title="LGTM">ampie</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-plugin-jpseo).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-plugin-jpseo/blob/master/OWNERS)~~ [ampie]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
